### ISSUE TITLE
Why did GPT-2 not use dropout in the model?

### ISSUE THREAD

**Issue by Charrnander** [openai/gpt-2#120]
I noticed that there is no dropout applied in the gpt-2 model, but a dropout of 0.1 was applied in  Bert.  Is there no overfiting problem in training gpt-2?

**Comment by 8enmann** [comment]
Dropout isn't used at test time; there's no training code provided here
